Makefile

```diff
@@ -70,23 +79,24 @@ dev: DEBUG = desktop:*
 dev: 
 	$(MAKE) start NODE_ENV=$(NODE_ENV) DEBUG=$(DEBUG)
 
-
 BASE_CONFIG := $(THIS_DIR)/desktop-config/config-base.json
-ENV_CONFIG := $(THIS_DIR)/desktop-config/config-$(CONFIG_ENV).json
+TARGET_CONFIG := $(THIS_DIR)/desktop-config/config-$(CONFIG_ENV).json
 
-.PHONY: desktop$/config.json
-desktop$/config.json:
-ifeq (,$(wildcard $(ENV_CONFIG)))
+.PHONY: desktop/config.json
+desktop/config.json:
+ifeq (,$(wildcard $(TARGET_CONFIG)))
 	$(warning Config file for environment "$(CONFIG_ENV)" does not exist. Ignoring environment.)
 else
 	$(eval EXTENDED = true)
 endif
-	@node -e "const base = require('$(BASE_CONFIG)'); let env; try { env = require('$(ENV_CONFIG)'); } catch(err) {} console.log( JSON.stringify( Object.assign( base, env ), null, 2 ) )" > $@
+	@node -e "const base = require('$(BASE_CONFIG)'); let env; try { env = require('$(TARGET_CONFIG)'); } catch(err) {} console.log( JSON.stringify( Object.assign( base, env ), null, 2 ) )" > $@
 	
-	@echo "$(GREEN)$(CHECKMARK) Config built $(if $(EXTENDED),(extended: config-$(CONFIG_ENV).json),)$(RESET)"
+	@echo "$(CYAN)$(CHECKMARK) Config built $(if $(EXTENDED),(extended: config-$(CONFIG_ENV).json),)$(RESET)"
 
 # Build calypso bundle
 build-calypso: 
+	@echo "Building calypso..."
```

Comment:
just stylistically speaking if we don't need colors we can use `$(info …)` and `$(warning …)` - the `@echo` invocation creates a new shell process but the macros/functions output from within the `make` process itself

Reply:
Good call.  Changed in d3e158f.

Will probably do another pass over the entire Makefile in a separate PR.